examples/nginx-plus-ingress-scrape-config.yaml

```diff
@@ -2,10 +2,10 @@
   kubernetes_sd_configs:
     - role: pod
   relabel_configs:
-    - source_labels: [__meta_kubernetes_pod_annotation_nsm_nginx_com_enable_ingress]
+    - source_labels: [__meta_kubernetes_pod_label_nsm_nginx_com_enable_ingress]
       action: keep
       regex: true
-    - source_labels: [__meta_kubernetes_pod_annotation_nsm_nginx_com_enable_egress]
+    - source_labels: [__meta_kubernetes_pod_label_nsm_nginx_com_enable_egress]
```

Comment:
I think we should still support annotations in the scrape config until they're removed, in the case that someone is still using annotations and happens to download this config.

Reply:
We would need to have a copy of this scrape job for annotations. All of the referenced source labels have to be present otherwise the metrics will not be scraped. This is what tripped me up when testing this change: I had both annotations and labels referenced in the same job so the metrics didn't show up.

Do you think it's worth keeping both since these docs won't be published until the label support is added? Since we have marked the annotation as deprecated, I'd prefer to keep references to them at a minimum.

Reply:
Oh, didn't realize that it's all or nothing. That's fair then, we can just use labels.